Workbooks/Storage/Overview/Overview.workbook

```diff
@@ -44,16 +44,19 @@
             "delimiter": ",",
             "query": "summarize by subscriptionId\r\n| project value = strcat(\"/subscriptions/\", subscriptionId), label = subscriptionId, selected = iff(subscriptionId =~ '{DefaultSubscription_Internal}', true, false)\r\n",
             "crossComponentResources": [
-              "value::selected"
+              "value::all"
             ],
             "isHiddenWhenLocked": false,
             "typeSettings": {
-              "limitSelectTo": 10,
+              "limitSelectTo": 25,
               "additionalResourceOptions": []
             },
             "timeContextFromParameter": null,
             "queryType": 1,
-            "resourceType": "microsoft.resourcegraph/resources"
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": [
```

Comment:
this actually has a selected subscription inside, should be value::selected or all or whatever, no specific resources should be selected

Reply:
Fixed in new PR